scripts/write-config.sh

```diff
@@ -0,0 +1,201 @@
+#!/bin/bash
+#
+# write-config.sh extracts configuration information from vault and writes it to a set of files
+# in a directory. This simplifies access to the secrets from other scripts and applications.
+#
+# We want to use this in a gradle task, so it takes arguments both as command line
+# options and as envvars. For automations, like GHA, the command line can be specified.
+# For developer use, we can set our favorite envvars and let gradle ensure the directory is properly populated.
+#
+# The environment passed in is used to configure several other parameters, including the target for running
+# the integration tests.
+#
+# For personal environments, we assume that the target name is the same as the personal namespace name.
+# The output directory includes the following files:
+#   ---------------------------+-------------------------------------------------------------------------
```

Comment:
Not sure if everething in write-config.sh is currently needed. I removed the configs for database creds. I assume we'll need the test runner configs once we set up integration test github workflow is set up?